cmd/evm/testdata/25/exp.json

```diff
@@ -0,0 +1,38 @@
+{
+  "alloc": {
+    "0x8a8eafb1cf62bfbeb1741769dae1a9dd47996192": {
+      "balance": "0x1"
+    },
+    "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b": {
+      "balance": "0x5ffd4878bc29ed73",
+      "nonce": "0xad"
+    },
+    "0xc94f5374fce5edbc8e2a8697c15331677e6ebf0b": {
+      "balance": "0x520800"
+    }
+  },
+  "result": {
+    "stateRoot": "0x36612be4ad9b2b2894cbf6ede6a7d27da8adaabdbb7f2cfe4d00a2bac72c7bd7",
+    "txRoot": "0x572690baf4898c2972446e56ecf0aa2a027c08a863927d2dce34472f0c5496fe",
+    "receiptsRoot": "0x056b23fbba480696b65fe5a59b8f2148a1299103c4f57df839233af2cf4ca2d2",
+    "logsHash": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+    "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+    "receipts": [
+      {
+        "root": "0x",
+        "status": "0x1",
+        "cumulativeGasUsed": "0x5208",
+        "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+        "logs": null,
+        "transactionHash": "0x92ea4a28224d033afb20e0cc2b290d4c7c2d61f6a4800a680e4e19ac962ee941",
+        "contractAddress": "0x0000000000000000000000000000000000000000",
+        "gasUsed": "0x5208",
+        "blockHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+        "transactionIndex": "0x0"
+      }
+    ],
+    "currentDifficulty": null,
+    "gasUsed": "0x5208",
+    "currentBaseFee": "0x500"
```

Comment:
I think this should be different than `0x500` though since the parent was `0x500`?

Reply:
Yeah it's returning early in `misc.CalcBaseFee`. For some reason the JSON isn't being properly parsed.

Reply:
test_cmd.go:261: (stderr:1) ParentGasUsed: 0
    test_cmd.go:261: (stderr:1) ParentGasLimit: 0

Both are zero
```golang
	if parent.GasUsed == parentGasTarget {
		return new(big.Int).Set(parent.BaseFee)
	}
```

Reply:
hah
```
		ParentGasUsed    *math.HexOrDecimal64                `json:"parentGasUsed,omitempty"`
		ParentGasLimit   *math.HexOrDecimal64                `json:"parentGasUsed,omitempty"`
```

Reply:
Did you manually modify an auto-generated file?

Reply:
I pushed the fixed autogenerated file (`go generate execution.go`) now. I'll leave it to you to fix up failing tests :)